lib/utils/responsive/responsive_utils.dart

```diff
@@ -99,5 +99,7 @@ class ResponsiveUtils {
     }
   }
 
+  bool isWidthMobile(BuildContext context) => getDeviceWidth(context) < minDesktopWidth;
```

Comment:
please dont missing the configuration like that. `less` than `desktop` is `tablet` not `mobile`. If you do that, in the future we will be hard to apply tablet.

Reply:
at the moment, please use `breakpoint1`, `breakpoint2`, .....